test/Microsoft.AspNetCore.Diagnostics.FunctionalTests/StatusCodeSampleTest.cs

```diff
@@ -31,5 +32,46 @@ public StatusCodeSampleTest(TestFixture<StatusCodePagesSample.Startup> fixture)
             Assert.Equal(HttpStatusCode.OK, response.StatusCode);
             Assert.Contains("Status Code: 417", body);
         }
+
+        [Fact]
+        public async Task StatusCodePageOptions_ExcludesSemicolon_AndReasonPhrase_WhenReasonPhrase_IsUnknown()
+        {
+            //Arrange
+            var httpStatusCode = 541;
+            var request = new HttpRequestMessage(HttpMethod.Get, $"http://localhost/?statuscode={httpStatusCode}");
+
+            //Act
+            var response = await Client.SendAsync(request);
+
+            var statusCode = response.StatusCode;
+            var statusCodeReasonPhrase = ReasonPhrases.GetReasonPhrase(httpStatusCode);
+
+            var responseBody = await response.Content.ReadAsStringAsync();
+
+            //Assert
+            Assert.Equal((HttpStatusCode)httpStatusCode, response.StatusCode);
+            Assert.DoesNotContain(";", responseBody);
+        }   
+
+        [Fact]
+        public async Task StatusCodePageOptions_IncludesSemicolon__AndReasonPhrase_WhenReasonPhrase_IsKnown()
+        {
+            //Arrange    
+            var httpStatusCode = 400;
+            var request = new HttpRequestMessage(HttpMethod.Get, $"http://localhost/?statuscode={httpStatusCode}");
+
+            //Act
+            var response = await Client.SendAsync(request);
+
+            var statusCode = response.StatusCode;
+            var statusCodeReasonPhrase = ReasonPhrases.GetReasonPhrase(httpStatusCode);
+
+            var responseBody = await response.Content.ReadAsStringAsync();
+
+            //Assert
+            Assert.Equal((HttpStatusCode)httpStatusCode, response.StatusCode);
+            Assert.Contains(";", responseBody);
+            Assert.Contains(statusCodeReasonPhrase, responseBody);
```

Comment:
I wanted to try and use the value in some way, but all it will be doing is confirming an empty string exists in the output (as that's the value returned from `ReasonPhrases.GetReasonPhrase`).

I don't see any value in this so I will remove the check from the test, as the semicolon check is sufficient in conjunction with the other test.